conditions-reference_5cece6cc2c7d3a32d2a4dac4.md

```diff
@@ -121,6 +156,130 @@ identified with one of the `operators` from above are executed with those values
 
 \* PCRE = Perl Compatible Regular Expression
 
+## Functions
+
+The functions allow you to perform certain more complex checks that are not just
+direct boolean or regex matches.
+Next is the list of currently supported functions.
+
+### change_in
+
+*Note*: This feature is currently in `beta` stage of development.
+
+The `change_in` function accepts one path or list of paths within the repository
+as a first parameter and for a particular range of commits it evaluates as true
+if any of the files that were changed matches one of the given paths.
+
+``` txt
+change_in(<path>, [configuration])
+or
+change_in(<paths>, [configuration])
+
+<path>  - Required, string with the absolute or relative path within the repository.
+          Relative paths are relative to the location of the yml file.
+          e.g. 'svcA/lib' or '/doc'
+
+<paths> - Required, list of strings with absolute or relative paths within the
+          repository. Relative paths are relative to the location of yml file.
+          e.g. ['svcA/lib', '/doc', '../README.md']
+
+[configuration] - Optional, map containing the values for configurable parameters.
+                  All parameters and their default values are stated bellow.
+                  e.g. {on_tags: false, default_branch: 'master-new'}
+```
+
+The `change_in` function calculates the commit range that should be examined in
+different ways depending on whether the workflow was initiated from the master
+branch or some other branch, or if it is a tag or a pull request.
+
+The default behavior is the following:
+
+- On `master` branch the examined commit range is all the commits within the push
+that initiated the workflow.
+The same range is available in a job environment as an environment variable called
+`SEMAPHORE_GIT_COMMIT_RANGE`.
+The changes that are compared to the paths given as parameters in this case are
+equivalent to the result of `git diff <sha 1>^ <sha N>` command where `sha 1` is
+the first commit of the push and the `sha N` is the last.
+
+- On `other branches` the examined commit range is wider and all the commits
```

Comment:
By putting the "other branches" in code blocks you are giving the impression that this is a concreate value to be used in the configuration.

I would also recommend "On other (non-master) branches the..."